src/utilities.ts

```diff
@@ -70,10 +70,20 @@ export const fillUsersTable = (users: Array<User>) => {
         const dateOfBAsStr = (user.yearOfBirth).toString()
         const pavelDVisitsAsStr = (user.pavelDurovVisits).toString()
         const idAsStr = users.indexOf(user)
-        table.innerHTML += table.appendChild(document.createElement("tr")).innerHTML =
-            '<td class="col s3">' + userName + "</td>" +
-            '<td class="col s3">' + idAsStr + "</td>" +
-            '<td class="col s3">' + dateOfBAsStr + "</td>" +
-            '<td class="col s3">' + pavelDVisitsAsStr + "</td>"
+        if (user.surname) {
+            table.innerHTML += table.appendChild(document.createElement("tr")).innerHTML =
```

Comment:
Please separate these actions into more readable format and supply documentation for them (and same for similar code below). e.g.
```js
// Create new table row element to fill with content
const tableRow = document.createElement("tr")
// Fill it with content
tableRow.innerHTML = ...
```
and so on. In general, provide documentation (via comments) for your thinking. Good rule of thumb is to document the intent, but not the action itself (which is obvious). For example:
```js
// Poor comment:
// Add 7 to b
const a = b + 7

// Good comment:
// Increase b by an amount of bits we cached during saving
const a = b + 7
```